main.js

```diff
@@ -989,21 +995,39 @@ function _select_Gateway() { // Check for Gateway used to
         settings["charcount"] = 99;
     }
 
+	var tasklist;
+	// Profession priority list by order
+	var defaultTasklist = [

```

Comment:
List  order can be changed.

Option gives easy way arrange items, I think "Events" can be bottom other list I revert to "old order".